packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -83,11 +83,9 @@ export const BookingFields = ({
           return null;
         }
 
-        // Attendee location field can be edited during reschedule
         if (field.name === SystemField.Enum.location) {
-          if (locationResponse?.value === "attendeeInPerson" || "phone") {
-            readOnly = false;
-          }
+          // TODO: We need to ensure that location field is properly updated in Booking Success Page, email, calendar, webhook after reschedule
+          readOnly = false;
```

Comment:
Made this change because `if (locationResponse?.value === "attendeeInPerson" || "phone") {` condition is faulty. 

It is always truthy. So, it seems like location change is enabled for attendeeInPerson and phone but in fact it is enabled in all cases.

Introduced in #11651



I think with this PR we should intend to be able to make any change to location field and test it accordingly.